src/core/Colors/Colors.baseStyles.tsx

```diff
@@ -41,7 +45,7 @@ export const baseStyles = withSuomifiTheme(
         }
 
         &--key {
-          ${theme.typography.bodySemiBold}
+          ${internalTypographyTokens.bodySemiBold}
```

Comment:
I assume this is not yet implemented / added to theme, so we use an internal set instead?